packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -65,17 +65,24 @@ const chatMessageStyles: ComponentSlotStylesInput<
   }),
 
   author: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    marginTop: pxToRem(5),
```

Comment:
I had to add this here in order to resolve some screen regressions, but I am pretty sure this will produce some differences on client's side. Not sure why this happened...